### PR TITLE
services/ntp - detach limited from kod, correct help text

### PR DESCRIPTION
**Pull request by ZsZs73**
Hi @fichtner,
This is the PR to address [issue #5470](https://github.com/opnsense/core/issues/5470)
I am not a regular github user so I hope that you can extract something useful from this PR.

### CODE REVIEW

**Comment by ZsZs73** [comment]
Forgot to ask how to make sure that `limited` is unchecked by default (on new installs).
Note that this would change the default behaviour as previously it was set to `limited`.

File: src/www/services_ntpd.php [hunk 1]
```diff
@@ -400,11 +401,14 @@ function removeRow() {
                       <input type="button" id="showrestrictbox" class="btn btn-default btn-xs" value="<?= html_safe(gettext('Advanced')) ?>" /> - <?=gettext("Show access restriction options");?>
                       </div>
                       <div id="showrestrict" style="display:none">
-                      <?=gettext("These options control access to NTP from the WAN."); ?>
+                      <?=gettext("These options control access to NTP from all interfaces."); ?>
```
Comment (fichtner) [inline src/www/services_ntpd.php:404]:
```suggestion
                      <?=gettext("These options control access to NTP."); ?>
```

Might be better to make it shorter?

**Comment by fichtner** [comment]
merged, thanks! Will probably land in 22.1 :)

**Comment by ZsZs73** [comment]
Thank you, however I am still not sure if `limited `will be enabled by default or not.

**Comment by fichtner** [comment]
Did a test and "limited" renders in the config and box is checked by default. Unchecking saves it, removes "limited" from config. Checking it again adds "limited" and removes setting from config... works fine from what I can tell :)

**Comment by ZsZs73** [comment]
I've also tesed it like you did before submitting the PR. 
I meant if the `limited` will be enabled on a fresh install or not. I could not find how the initial `ntpd.conf` gets generated after a fresh install. Default `ntpd.conf` does not exist in the ntp package.
IMHO `limited` should not be enabled on a fresh install like it is not enabled on server OSs.

**Comment by fichtner** [comment]
It is added on a fresh install to retain backwards compatibility. I thought that was your intention.  ;)

**Comment by ZsZs73** [comment]
I realized that there is a rate limit in place because some of my IoT devices could not sync to my OPNsense firewall wich had default NTP config with `limited `enabled.
In my opinion th default ntpd config with `limited `enabled is too restrictive for a local network, espescially for some IoT devices (namley ones running [NTPClient](https://github.com/arduino-libraries/NTPClient))
However I confirm that other linux boxes can sync even with iburst set.

To summarize:
- `limited` enabled: 
  - IoT devices might not be able to sync (two NTP requests immediately after each other)
  - Linux boxes with full blown ntpd running on them sync well even with `iburst` set. (NTP requests with 2 secs pause between them)
- `limited` disabled: anything I checked could sync properly

In meanwhile I'll check if the NTPClient can be fixed, but it will not help to the already deployed IoTs which possibly never get updated.

**Comment by fichtner** [comment]
But leaving limited out of the default configuration will likely back off hardening efforts. I don’t think IoT is a good reason to roll back such protection.
